src/analysis/normed_space/operator_norm.lean

```diff
@@ -395,24 +395,21 @@ begin
     rw [dist_eq_norm, dist_eq_norm, ← f.map_sub, H] }
 end
 
-lemma homothety_norm (hE : 0 < vector_space.dim 𝕜 E) (f : E →L[𝕜] F) {a : ℝ} (ha : 0 ≤ a) (hf : ∀x, ∥f x∥ = a * ∥x∥) :
+lemma homothety_norm [nontrivial E] (f : E →L[𝕜] F) {a : ℝ} (ha : 0 ≤ a) (hf : ∀x, ∥f x∥ = a * ∥x∥) :
```

Comment:
You don't need the assumption that `a` is nonnegative, as it can be deduced from `hf` applied to a nonzero `x`.

Reply:
I didn't write that one, but nice catch!